app/src/main/java/org/dhis2/Bindings/StringExtensions.kt

```diff
@@ -64,3 +64,5 @@ fun String.toDate(): Date {
 
     return date
 }
+
+fun String.parseToDouble() = this.toDoubleOrNull()?.toString() ?: "0.0"
```

Comment:
I think it would be a nice idea to validate somehow -90/90 and -180/180. Otherwise that could be upload to the server and when entering the dashboard get an alert dialog about wrong coordinates. What do you think?

Reply:
done!